src/eicr_anonymization/anonymize_eicr.py

```diff
@@ -202,7 +201,7 @@ def anonymize(args: Namespace) -> None:
     if args.command == "debug":
         debugOptions = DebugOptions(args.seed, args.deterministic_functions)
     anonymizer = Anonymizer(debugOptions)
-    parser = Parser(patient_only=args.patient_only)
+    parser = Parser(patient_only=args.config)
```

Comment:
```suggestion
    parser = Parser(config=args.config)
```
can we rename the parser field as well 🙏